api/v1alpha1/kafka.go

```diff
@@ -1,7 +1,8 @@
 package v1alpha1
 
 type KafkaNET struct {
-	TLS *TLS `json:"tls,omitempty" protobuf:"bytes,1,opt,name=tls"`
+	TLS  *TLS  `json:"tls,omitempty" protobuf:"bytes,1,opt,name=tls"`
+	SASL *SASL `json:"sasl,omitempty" protobuf:"bytes,2,opt,name=sasl"`
```

Comment:
how come `sasl` goes under `net`? I'd expect it to be under a `auth` node. You can only have TLS or SASL - is that correct - I can't mix them?

Reply:
It's under `net` in sarama lib as well.

Per the docs, `TLS` could be used for encryption or authentication, `sasl` only for authentication, they seem to be able to be used together, one for encryption another one for auth, even though I couldn't find any evidence from internet.

Reply:
ok, that's a good reason, lets have it under net